docs/user_manual/calculations.md

```diff
@@ -125,6 +125,26 @@ Output:
 - Node voltage magnitude and angle
 - Current flowing through branches and fault.
 
+#### Common calculations
+
+Power flowing through a branch is calculated by voltage and current for any type of calculations in the following way:
+
+$$
+    \begin{eqnarray}
+        \underline{S_{branch-side}} = \sqrt{3} \cdot \underline{U_{node-line-line-side}} \cdot \underline{I_{branch-side}}
+    \end{eqnarray}
+$$
+
+These quantities are in complex form. Hence, they can be constructed by PGM output attributes in the following way:
+
+For  $\underline{U}$ of nodes, `u`, `u_angle` can be used.
+
+For  $\underline{I}$ of branches, `i_side`, `p_side`, `q_side` can be used: $\underline{I} = i_{side} \angle \arctan(((p_{side} + j \cdot q_{side}) / \underline{U})*)$.
```

Comment:
a lot of nested braces. can you use `\frac` to pretty it up a little bit? also:
* please double-check that the `*` works as intended, it's not superscripted `^*`
* the `\angle` feels out-of-place here. `\arctan` already implies `\angle`. or you can just make it `\exp i ...`. even if I understand what you mean, at the very least, it's a notation i've never seen before.